rpcs3/Input/pad_thread.cpp

```diff
@@ -197,6 +206,12 @@ void pad_thread::Init()
 			{
 				input_log.notice("Pad %d: config=\n%s", i, cfg->to_string());
 			}
+
+			// If the user changes the emulated controller, then simulate unplugging and plugging in a new controller
+			if ((pad_settings[i].class_type != pad->m_class_type || pad_settings[i].vendor_id != pad->m_vendor_id || pad_settings[i].product_id != pad->m_product_id) && m_pads_connected[i])
```

Comment:
this doesn't really make sense, since it also applies to any newly connected controller if I'm correct.

Reply:
A newly connected controller wouldn't have m_pads_connected[i] be true would it?

Reply:
I guess you are right.
But then please move that crucial condition to the front of the check.

Reply:
done